### PR TITLE
chore: mantine-fy dashboard update modal

### PR DESCRIPTION
**Pull request by joaoviana**
Relates to: #8301 

### Description:

dashboard update modal

before

Screenshot 2023-12-13 at 16 21 13

after

Screenshot 2023-12-13 at 16 20 38

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/common/modal/DashboardUpdateModal.tsx [hunk 1]
```diff
@@ -31,69 +31,73 @@ const DashboardUpdateModal: FC<DashboardUpdateModalProps> = ({
...
+        >
+            <form title="Update Dashboard" onSubmit={handleConfirm}>
+                <Stack spacing="lg" pt="sm">
+                    <TextInput
                         label="Enter a memorable name for your dashboard"
-                        name="name"
+                        required
                         placeholder="eg. KPI Dashboards"
                         disabled={isUpdating}
-                        rules={{ required: 'Name field is required' }}
```
Comment (IrakliJani) [inline packages/frontend/src/components/common/modal/DashboardUpdateModal.tsx:66]: now that we don't have `rules` will the validation still work?

**Review by IrakliJani — APPROVED** [review]
just one question. I'm checking render now

**Review by IrakliJani — APPROVED** [review]
I guess this is enough!

CleanShot 2023-12-14 at 15 56 24@2x

File: packages/frontend/src/components/common/modal/DashboardUpdateModal.tsx (hunk 1 above)
Comment (joaoviana) [reply packages/frontend/src/components/common/modal/DashboardUpdateModal.tsx:66]: yes because it's `required`! 👍

**Pull request merged**